src/main/channels/experiment/PostOneExperimentChannel.ts

```
@@ -179,7 +179,7 @@ export class PostOneExperimentChannel extends PostOneChannel {
                   exchangeTemplate.participantInstructionsOnComplete;
                 exchange.cue = exchangeTemplate.cue;
                 exchange.order = exchangeTemplateNumber;
-                exchange.triggers = exchangeTemplate.triggers;
+                exchange.triggers = exchangeTemplate.triggers ?? [];
```

Add a comment to explain why this operator is needed.

---

I used this so exchange.triggers is always assigned an array because if exchangeTemplate.triggers is null or undefined, the bot would silently fail and never respond in the conversation.

---

Sounds good. Maybe add a comment above it:

```
// exchange.triggers is always assigned an array because if exchangeTemplate.triggers
// is null or undefined, the bot would silently fail and never respond in the conversation
```